src/Differ.php

```diff
@@ -40,6 +45,14 @@ public function __construct($header = "--- Original\n+++ New\n")
     }
 
     /**
+     * @param bool $show
+     */
+    public function setShowNonDiffLines($show)
```

Comment:
Is a setter method really what we should have. What about adding a 2nd param to __construct.